bibliopixel/animation/animation.py

```diff
@@ -119,7 +119,8 @@ def run_context(self):
 
         self.check_delay()
 
-        self.preRun(self.runner.amt)
+        self.layout.all_off()
```

Comment:
100% agreed. That is why I will remove the all_off() from here so that it's entirely up to each animation. If an animation wants to clear the screen, it's up to it to do so. pre_run will also be where anything else you need initialized every time it starts happens of course. But don't worry, all_off() is going away.